RxJavaTemplate/app/src/main/java/co/nimblehq/rxjava/ui/base/BaseFragment.kt

```diff
@@ -69,7 +77,8 @@ abstract class BaseFragment : Fragment(), BaseFragmentCallbacks {
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
-        return inflater.inflate(layoutRes, container, false)
+        _binding = bindingInflater.invoke(inflater, container, false)
+        return requireNotNull(_binding).root
```

Comment:
we could rewrite this a bit to avoid using `requireNotNull`
```suggestion
        return bindingInflater.invoke(inflater, container, false).apply {
            _binding = this
        }.root
```